src/config.py

```diff
@@ -14,5 +18,15 @@ def update_config():
   with open('config/2019.json', 'r') as config_file:
     config_json['2019'] = json.load(config_file)
 
+    for contributor in config_json['2019']['contributors'].items():
+      if 'gravatar' not in contributor[1] and 'teams' in contributor[1]:
```

Comment:
Contributors without a gravatar will have deterministically-generated default avatars from [this set](https://drive.google.com/file/d/1p0mh0hcHLoY19G7CHKVEkYHI_jdLJvmT/view?usp=sharing). We still need to cookie-cut each graphic out, but for now could you change this logic from team-based avatars to deterministically-generated? Let's just rename the team-based images to `images/avatars/1.jpg` and keep a constant in this file for the number of default avatars.

For example, hash the contributor ID and take the modulo to select a particular avatar.

Also, it might be cleaner to handle the case when contributors _do_ have a gravatar first, then in the `else` block select the default avatar.